explainability-core/src/main/java/org/kie/trustyai/explainability/model/Dataframe.java

```diff
@@ -364,6 +408,7 @@ public List<Value> getRow(int row) {
     public List<List<Value>> getRows() {
 
         return rowIndexStream()
+                .parallel()
```

Comment:
Can you please clarify the usage of `parallel` in this class? It can produce tricky outcome (by default it uses a number of thread that takes all CPU available).
Is it to improve performance in general?

Reply:
Hi @danielezonca this was a left-over from some test with parallel access. I believe we can implement access/processing parallelisation for the dataframes, but I agree it should be more thought about and in a separate PR, perhaps.